corehq/apps/app_manager/templates/app_manager/partials/releases.html

```diff
@@ -65,8 +65,10 @@
                 {% include 'app_manager/partials/releases_table.html' %}
             </div>
             {% if enable_update_prompts %}
+                 <script src="{% static 'app_manager/js/releases/update_prompt.js' %}"></script>
```

Comment:
This should go in `app_manager/app_view.html`. We're moving away from using script tags and towards using requirejs, and that migration is a little bit easier if all the script tags are in the `js` blocks in the big base templates.